README.md

```diff
@@ -1 +1,151 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+  <li><a href="#">HTML</a></li>
+  <li><a href="#">CSS</a></li>
+    <li><a href="#">JavaScript</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add Lists**
+- **Remove Completed Lists**
+<p align="right">
+(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- comming soon
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+  ## 💻 Getting Started <a name="getting-started"></a>
+
+  To get a local copy up and running it in your machine, follow the next steps.
+
+  ### Prerequisites
+
+  In order to run this project you need:
+
+- Internet connection.
+- Basic knowledge of Git/Terminal.
+- Visual Studio Code Installed.
+
+### Setup
+
+Clone this repository to your desired folder
+
+### Install
+
+Install almost all dependencies.
+This one will install stylelinter:
+
+> npm install --save-dev
+> stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+To run the project, execute in live server.
+
+## 👥 Author <a name="author"></a>
+
+👤 **Haftamu Desta**
+
+- GitHub: [@haftamu](https://github.com/haftamudesta)
+- Twitter: [@DestaHaftamu](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
+- LinkedIn: [Haftamu Desta](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [] **display scores**
+- [] **submit scores**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Hi, I would appreciate if you give this project a star(⭐️) this will encourage me to improve it and build the social network of the future.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> "Never stop learning, code never stop teaching"
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Can I publish this project as mine?**
+
+  - No, but you can use it as an inspiration or for educational purposes.
+
+- **Can I modify it while practicing?**
+
+  - Yes, you can do it, but make sure that you know what are you doing, we never know when we can get stuck at some point just because we were playing with the code _-laughs-_
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] You did a good job working on your project documentation! 💯 However, please ensure all the links are working, this one is looking for a LICENSE file in your repo but you don't have one yet. Microverse suggests that you use the MIT one, you could set it following the steps from [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).